### ISSUE TITLE
Design a footer section where people can subscribe

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/website-old#62]
Now that the subscribe link is not on the home page https://github.com/luckyframework/website/pull/61

**Comment by jwoertink** [comment]
subscribe for what? There's a footer now, so if you want some email form, it can be added. However, do you really want to manage a newsletter? 😛

**Comment by paulcsmith** [comment]
Yeah I’d like to add something where you can subscribe to updates but this is super low priority right now. Lots of more important stuff to do!
